docs/notebooks/nmf_tutorial.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Missing:
- how to infer vector for word \ document
- description of internal state (matrices, where\what stored)
- benchmark table \ plot
- (?) Keras example?

Reply:
Still missing/need to fix
- what we have in NMF in addition (normalization for using it as TM), 
- "Training" header duplicated
- Can you be a bit more "descriptive" (not only headers, also text descriptions)
- Benchmark require some conclusions (not just show a table, here you have many metrics & many parameters)
- Parameter tuning
- What happens with gensim NMF faces in notebook?